app/db/migrations/20171016091057-create-product.js

```diff
@@ -55,10 +52,14 @@ module.exports = {
       shop_id: {
         type: Sequelize.INTEGER
       },
-      brands_id: {
+      downloadable: {
+        type: Sequelize.BOOLEAN,
+        defaultValue: false
+      },
+      brand_id: {
```

Comment:
I think uniformity is important, if we are going to make use of `UUID` or `Integer` we should make use of it throughout the ids.

Reply:
There is uniformity, integer is what we're using